recipes/emperor/meta.yaml

```diff
@@ -0,0 +1,51 @@
+# Jinja variables help maintain the recipe as you'll update the version only here.
+{% set version = "3.8.2" %}
+
+package:
+  name: emperor
+  version: {{ version }}
+
+source:
+  fn: emperor-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/bd/12/52fb3ac9bcdd7dd34fe6617c23d493fc42ae637bc71dfaef875e05bffbbe/{{ version }}.tar.gz
```

Comment:
The preferred URL format (more readable) is:

https://pypi.io/packages/source/e/emporer/emporer-{{ version }}.tar.gz